spec/models/atmosphere/virtual_machine_spec.rb

```diff
@@ -323,6 +323,22 @@ def delete_with_success_when_exception(e)
     end
   end
 
+  context '::monitorable' do
+    let!(:vm) { create(:virtual_machine, ip: priv_ip) }
+    let!(:managable_vm) { create(:virtual_machine, managed_by_atmosphere: true,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.